cpg_infra/billing_aggregator/driver.py

```diff
@@ -242,20 +258,8 @@ def setup_gcp_cost_reporting(self):
         )
 
         # Create the http target to trigger the cloud function directly
-        http_function_target = (
-            gcp.cloudscheduler.JobHttpTargetArgs(
-                uri=function.service_config.uri,
-                http_method='POST',
-                headers={
-                    "Content-Type": "application/x-www-form-urlencoded",
-                },
-                oidc_token=gcp.cloudscheduler.JobHttpTargetOidcTokenArgs(
-                    audience=function.service_config.apply(
-                        lambda service_config: f"{service_config.uri}/",
-                    ),
-                    service_account_email=service_account,
-                ),
-            ),
```

Comment:
I imagine this failed because of this trailing `,` inside of parenthesis, turning this into a tuple of length 1.